neuro_extras/image.py

```diff
@@ -157,42 +189,45 @@ async def _image_transfer(src_uri: str, dst_uri: str, force_overwrite: bool) ->
         dockerfile.write_text(
             textwrap.dedent(
                 f"""\
-                FROM {_as_repo_str(src_image)}
+                FROM {src_image.as_docker_url()}
                 LABEL neu.ro/source-image-uri={src_uri}
                 """
             )
         )
+        migration_job_tags = (
+            f"src-image:{str(src_image)}",
```

Comment:
minor: `str()` not necessary here, the cast to str will be done inside the f-string

Reply:
got you, thanks! 
removed in bfc2b15